i18n/zh-CN/docusaurus-plugin-content-docs/current/reference/model/file.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Could you please add documents for the file module here for i18n? https://github.com/kcl-lang/kcl-lang.io/blob/main/i18n/zh-CN/docusaurus-plugin-content-docs/current/reference/model/overview.md

Reply:
fixed